requirements.txt

```diff
@@ -8,6 +8,9 @@ torch
 num2words==0.5.10
 word2number==1.1
 
+# for PhoneticMatchFilter
+jellyfish==0.8.2
```

Comment:
This should be in a separate requirements.txt in the folder `soundex`

Reply:
Resolved in 26eed69dba6909b7a04b2d544a3b92834c7b74a2